python/tests/conftest.py

```diff
@@ -304,11 +304,25 @@ def arctic_client_no_lmdb(request, encoding_version):
     assert not ac.list_libraries()
     return ac
 
+@pytest.fixture(
+    scope="function",
+    params=[
+        "lmdb"
+    ],
+)
+def arctic_client_lmdb(request, encoding_version):
```

Comment:
Don't parameterize this fixture, just pass in a `lmdb_storage` rather than the `request`